usr/src/uts/common/fs/zfs/dsl_scan.c

```diff
@@ -1329,21 +1813,22 @@ dsl_scan_visit(dsl_scan_t *scn, dmu_tx_t *tx)
 
 		if (spa_version(dp->dp_spa) < SPA_VERSION_DSL_SCRUB) {
 			VERIFY0(dmu_objset_find_dp(dp, dp->dp_root_dir_obj,
-			    enqueue_cb, tx, DS_FIND_CHILDREN));
+			    enqueue_cb, NULL, DS_FIND_CHILDREN));
 		} else {
 			dsl_scan_visitds(scn,
 			    dp->dp_origin_snap->ds_object, tx);
 		}
 		ASSERT(!scn->scn_pausing);
```

Comment:
We can pause when visiting block pointers (from `dsl_scan_visitbp()`).  The `dp_origin_snap` does not have any block pointers, so we can't pause while visiting it.  Visiting it serves only to add its "next clones" to the work queue.